src/test/java/org/hisp/dhis/rules/models/RuleActionHideOptionGroupTest.java

```diff
@@ -0,0 +1,45 @@
+package org.hisp.dhis.rules.models;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import nl.jqno.equalsverifier.Warning;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.assertj.core.api.Java6Assertions.fail;
+
+@RunWith(JUnit4.class)
+public class RuleActionHideOptionGroupTest
+{
+        @Test
+        public void createMustSubstituteEmptyStringIfArgumentsNull()
+        {
+            RuleActionHideOptionGroup ruleActionHideOptionGroup =
+                    RuleActionHideOptionGroup.create( null, "test_option_group" );
+
+            assertThat( ruleActionHideOptionGroup.content() ).isEqualTo( "" );
+        }
+
+        @Test
+        public void createMustThrowOnNullField()
+        {
+            try
+            {
+                RuleActionHideOptionGroup.create(  "test_content", null );
+                fail( "NullPointerException was expected, but nothing was thrown." );

```

Comment:
We are following a pattern of using @Rule annotation with ExpectedException to capture required Exception behavior. 
It's better if we can align this accordingly. Otherwise looks fine.